### ISSUE TITLE
`path` is undefined

### ISSUE THREAD

**Issue by smcelhinney** [zico4reel/React-Native-Firebase-implementation-fix#1]
Think you're missing an import for your config, ie.

```
const path = require('path');

module.exports = {
  dependencies: {
    'react-native-firebase': {
      platforms: {
        ios: { podspecPath: path.join(__dirname, 'ios', 'RNFirebase.podspec') },
        android: {
          /*"sourceDir": "./node_modules/react-native-firebase/android",
          "folder": "./node_modules/react-native-firebase",*/
          "packageImportPath": "import io.invertase.firebase.RNFirebasePackage;",
          "packageInstance": "new RNFirebasePackage()"
        }
      }
    }
  }
};
```

**Comment by ghost** [comment]
Same issue

**Comment by zico4reel** [comment]
Sorry for the issue and delayed response.

When I initially made the fix, it was android focus and the iOS part below wasn't considered,  in a rush to include for iOS so I can focus on the job, I added the line below, if you remove it, It works okay for Android dev

   ios: { podspecPath: path.join(__dirname, 'ios', 'RNFirebase.podspec') }

**Comment by zico4reel** [comment]
You can check this commit for the one that worked just fine for Android, I will be reverting back to it

https://github.com/zico4reel/React-Native-Firebase-implementation-fix/commit/26c738bd8cee16d7ab42e3f5f10b80e31d6d08eb
